dashboard/test/test_helper.rb

```diff
@@ -555,7 +555,7 @@ def storage_id(_)
 end
 
 def storage_id_for_user_id(user_id)
-  Random.new(user_id).rand(1_000_000)
+  Random.new(user_id.to_i).rand(1_000_000)
```

Comment:
is this needed, and if so why?

Reply:
shouldn't be. User.id is an [integer](https://github.com/code-dot-org/code-dot-org/blob/2ed5a622cdde08b270ebdb9480f0ab2044afda57/dashboard/app/models/user.rb#L5)

Reply:
I actually recommended this.  We ran into this in testing when `user_id` was still being passed as a URL parameter, so it came in as a string.  The normal `storage_id_for_user_id` method seemed to handle that just fine, but this test method required the provided `user_id` to be a number (because `Random.new` does).  The cast here seemed right because it makes the test method behave more like the real one.